src/components/overview/feedback/feedback.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Input, Label, List, Segment } from "semantic-ui-react";
+import { ApiService } from "../../../services/apiService";
+import AddFeedback from "../feedback/AddFeedback";
+const apiService = new ApiService();
+export default class Feedback extends React.Component<any, any> {
+    public state = {
+        feedback: "",
+        gottenFeedback: apiService.getFeedbackForDocumentIPFSHash(this.props.match.params.hash),
+    };
+    public handleChange = (e, data) => {
+        this.state.feedback = data.value;
+    };
+    public handleSubmit = (e, data) => {
```

Comment:
Remove unused parameters.

```suggestion
    public handleSubmit = () => {
```